Scheduler/feature_scheduler/maintel/fbs_config_minimal_fieldsurvey.py

```diff
@@ -94,20 +94,151 @@ def get_scheduler():
         # once SP-2080 functionality is available at summit.
     ]
 
+    observation_reason = "science"
+    science_program = "BLOCK-365"  # json BLOCK to be used
+
+    # Carina
+    # Custom landscape dither
+    radius = np.sqrt(1.8)
+    orientation = np.radians(-10.0)
+    delta_ra = [-1.0 * radius * np.cos(orientation), radius * np.cos(orientation)]
+    delta_dec = [-1.0 * radius * np.sin(orientation), radius * np.sin(orientation)]
+    config_detailers = [
+        detailers.DitherDetailer(max_dither=0.7, per_night=False),
```

Comment:
Why are you using the DitherDetailer AND the DeltaCoordDitherDetailer? They both offset the RA/Dec of the pointing. 
The DeltaCoordDitherDetailer is intended to give you a specific ra/dec grid of offsets, while the DitherDetailer will give you a random offset up to a given radius (max_dither) from the center.
The use-cases are usually somewhat orthogonal.

Reply:
This is exactly the functionality that I was hoping for -- small dithers around a set of pointing centers where we cycle through the pointing centers.